cargo-generate.toml

```diff
@@ -0,0 +1,7 @@
+[template]
+ignore = [
+    "contracts/minimal",
+    "contracts/cw20",
+    "contracts/cw721",
+    "contracts/cargo-generate.toml",
```

Comment:
Can these merge into one line?

Reply:
This is easier to read, so I'll leave it as is.